From 730984d472883b791e569b1440ff3f497cfe4b94 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 6 Mar 2019 08:03:39 -0500
Subject: [PATCH 1321/1936] Change Travis-CI Build To Use Distro Liblo Version

Ubuntu Xenial has version 0.28 of liblo, so we shouldn't need to
build it from source. This should resolve a SSL connection error
during the build. It's unclear if that error in particular is a
result of sourceforge having an underlying issue or a problem with
the SSL library available on travis's side.
---
 .travis.yml | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/.travis.yml b/.travis.yml
index 4f4a5175..7abd9463 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -7,12 +7,8 @@ dist: xenial
 
 before_install:
     - sudo apt-get install zlib1g-dev libmxml-dev libfftw3-dev dssi-dev libfltk1.3-dev libxpm-dev
+    - sudo apt-get install liblo-dev
     - sudo apt-get install --force-yes cxxtest
-    - wget http://downloads.sourceforge.net/liblo/liblo-0.28.tar.gz
-    - tar xvf liblo-0.28.tar.gz && cd liblo-0.28
-    - ./configure && make && sudo make install
-    - sudo ldconfig
-    - cd ..
 
 
 script:
-- 
2.47.0

